packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -4,50 +4,77 @@ import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
 
+import {
+  ChoiceMethod,
+  QuorumRule,
+  ElectionType,
+  looksLikeQuestionSpec,
+} from '../../../src/question.js';
+
+const { quote: q } = assert;
+
 const makeVoterVat = async (log, vats, zoe) => {
   const voterCreator = E(vats.voter).build(zoe);
   log(`=> voter vat is set up`);
   return voterCreator;
 };
 
-async function addQuestion(qDetails, closingTime, tools) {
+const createQuestion = async (qDetails, closingTime, tools, quorumRule) => {
   const { registrarFacet, installations } = tools;
-  const { question, positions } = qDetails;
+  const { issue, positions, electionType } = qDetails;
   const closingRule = {
     timer: tools.timer,
     deadline: 3n,
   };
 
-  const ballotDetails = {
-    question,
-    positions,
-    quorumThreshold: 3n,
-    tieOutcome: undefined,
-    closingRule,
-  };
-  const { instance: ballotInstance } = await E(registrarFacet).addQuestion(
-    installations.binaryBallotCounter,
-    ballotDetails,
+  const questionSpec = looksLikeQuestionSpec(
+    harden({
+      method: ChoiceMethod.UNRANKED,
+      issue,
+      positions,
+      electionType,
+      maxChoices: 1,
+      closingRule,
+      quorumRule,
+      tieOutcome: positions[1],
+    }),
   );
-  return ballotInstance;
-}
 
-async function committeeBinaryStart(
+  const { instance: counterInstance } = await E(registrarFacet).addQuestion(
+    installations.binaryVoteCounter,
+    questionSpec,
+  );
+  return { counterInstance };
+};
+
+const committeeBinaryStart = async (
   zoe,
   voterCreator,
   timer,
   log,
   installations,
-) {
+) => {
   const registrarTerms = { committeeName: 'TheCommittee', committeeSize: 5 };
   const { creatorFacet: registrarFacet, instance: registrarInstance } = await E(
     zoe,
   ).startInstance(installations.committeeRegistrar, {}, registrarTerms);
 
-  const choose = 'Choose';
-  const details = { question: choose, positions: ['Eeny', 'Meeny'] };
+  const choose = { text: 'Choose' };
+  const electionType = ElectionType.SURVEY;
+  const details = {
+    issue: choose,
+    positions: [harden({ text: 'Eeny' }), harden({ text: 'Meeny' })],
+    electionType,
+  };
+  const eeny = details.positions[0];
+  const meeny = details.positions[1];
```

Comment:
tweak for conciseness:

```suggestion
  const [eeny, meeny] = details.positions;
```

Reply:
Thanks. done.

I generally intend to use that style, so definitely point it out when I miss opportunities.